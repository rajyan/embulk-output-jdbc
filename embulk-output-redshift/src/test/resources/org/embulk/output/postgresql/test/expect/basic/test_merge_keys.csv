id:long,value1:string,value2:string
22,A2,b2
24,A4,b4
26,A6,b6
