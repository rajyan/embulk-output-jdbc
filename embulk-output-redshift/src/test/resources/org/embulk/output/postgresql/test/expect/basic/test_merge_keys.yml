table: test_merge
mode: merge
merge_keys:
- value1
