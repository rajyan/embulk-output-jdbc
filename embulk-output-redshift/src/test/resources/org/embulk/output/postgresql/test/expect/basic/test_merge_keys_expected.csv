11,A1,B1
13,A3,B3
22,A2,b2
24,A4,b4
26,A6,b6
